Description: Replace removed php_smart_string.h with zend_smart_string.h for lsphp85
Author: Your Name
Forwarded: not-needed
Last-Update: 2025-12-04

--- a/src/php7/php_igbinary.h
+++ b/src/php7/php_igbinary.h
@@ -34,7 +34,7 @@
 #endif

-#include "ext/standard/php_smart_string.h"
+#include "Zend/zend_smart_string.h"

 /** Module init function. */
 PHP_MINIT_FUNCTION(igbinary);
